lib/runner/extensions/event.command.js

```diff
@@ -198,6 +199,13 @@ module.exports = {
                     if (!err && (abortOnFailure || stopOnFailure)) {
                         err = postProcessContext(result);
                     }
+
+                    // Ensure that we have SDK instances, not serialized plain objects.
```

Comment:
Let's have a todo to update `postman-sandbox` to avoid serialization on every event trigger, this looks fine for now.